site/docs/components/form-field/examples.mdx

```diff
@@ -0,0 +1,189 @@
+
+- The label should be easy to scan and be no more than 3 words long.
+
+- Use the label to describe the information/data the field is collecting (e.g., Name, Location, Currency).
+
+- Use `FormFieldHelperText` to provide additional information and context to help the user complete the field. See the ‘Helper Text’ section below.
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="LabelLeft" >
+
+### Label placement left
+
+The form field label can be positioned to the left of the form control via the prop ‘labelPlacement’.
+
+- Place your label to the left of the control if your application is limited in vertical real estate or if you have a very data heavy, dense form where it’s preferable to have as many fields visible as possible without scrolling.
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="LabelQuestion" >
+
+### Label as Question
+
+Form Field labels can be styled with a different intent by setting the `intent` prop. Use “sentence” intent for sentences and questions. See more about [displaying labels](/salt/patterns/forms#label-vs-question).
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="MultipleChildren" >
+
+### Multiple Children
+
+You can display multiple controls within a single Form Field. Wrap the controls in a Stack Layout for the correct styling.
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="NecessityLabel" >
+
+### Necessity Label
+
+If a field is required, it can be marked with a "Required”, “Optional”, or “\*” indicator using the `necessity` prop.
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="Readonly" >
+
+### Readonly
+
+Form Field can be set as read-only via the prop ‘readOnly’.
+
+- The prop should be applied to the field, not the control it contains, for expected results.
+
+- Set the field to read-only if you’re displaying values that are predefined and not editable, e.g., entry that’s populated and fixed by template selection.
+
+- Adheres to WCAG AA text contrast for legibility; read-only field content is always relevant to the form, but cannot be changed.
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="Validation" >
+
+### Validation
+
+Form Fields can show validation states (warning, success, error) with the prop `validationStatus`.
+
+- The prop should be applied to the field,not the control it contains, for expected results.
+
+- Use the error state to alert the user of a critical issue that’s related to the field entry. This issue, which may jeopardize completion of the task, usually requires action from the user to resolve the error.
+
+- Display the warning state when you need to alert the user of a potential issue that won’t prevent the user from continuing with the task, but may cause errors if it’s not addressed.
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="WithControls" >
+
+### With Controls
```

Comment:
This section on its own could be merged with some others? maybe even "Default"..?